include/attribute_store.h

```diff
@@ -20,7 +20,7 @@
 
 struct AttributeStore
 {
-    using key_value_t = std::pair<std::string, vector_tile::Tile_Value>;    
+    using key_value_t = std::tuple<std::string, vector_tile::Tile_Value, char>;
```

Comment:
You can use a struct for this, instead of a tuple. That way you can name the fields, instead of std::get<0> std::get<1>